tools/pubsys/src/aws/ssm/template.rs

```diff
@@ -70,13 +70,31 @@ pub(crate) fn get_parameters(
     Ok(template_parameters)
 }
 
+/// A value which stores rendered SSM parameters alongside metadata used to render their templates
+#[derive(Debug, Eq, PartialEq, Hash, Clone)]
+pub(crate) struct RenderedParameter {
+    pub(crate) ami: Image,
```

Comment:
Can we just store the AMI-ID, or do we need everything else about the Image as well?

Reply:
We do currently only use the AMI ID, but I added the full AMI here so that the image could be used in the future while logging to provide context to any errors. Could be useful in cases where we want to understand why a template was rendered in a given way.

I'd be okay with removing it since we don't currently do that, but it seemed like a good idea to me.